ocp_resources/multi_network_policy.py

```diff
@@ -0,0 +1,57 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class MultiNetworkPolicy(NamespacedResource):
+    """
+    MultiNetworkPolicy object.
+
+    API reference:
+    https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-multi-network-policy.html
+    """
+
+    api_group = NamespacedResource.ApiGroup.K8S_CNI_CNCF_IO
+
+    def __init__(
+        self,
+        network_name=None,
+        policy_types=None,
+        ingress=None,
+        egress=None,
+        pod_selector=None,
+        **kwargs,
+    ):
+        """
+        Create and manage MultiNetworkPolicy
+
+        Args:
+            network_name (str): The name of the NetworkAttachmentDefinition that the policy will impact.
+            policy_types (list): One or both of the valid ip policies - "Ingress" and "Egress".
+            ingress (list): list containing a dictionary specifying the allowed "from" parameters.
+            egress (list): list containing a dictionary specifying the allowed "to" parameters.
+            pod_selector (dict): Map a label to match.
+        """
+        super().__init__(**kwargs)
+        self.network_name = network_name
+        self.policy_types = policy_types
+        self.pod_selector = pod_selector
+        self.ingress = ingress
+        self.egress = egress
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file:
+            if not self.network_name:
+                raise ValueError("network_name is required")
+
+            self.res["metadata"]["annotations"] = {
+                f"{self.api_group}/policy-for": f"{self.namespace} /{self.network_name}"
+            }
+            self.res["spec"] = {}
+            self.res["spec"].update({
+                "podSelector": {"matchLabels": self.pod_selector} if self.pod_selector else {},
```

Comment:
user should send the whole dict; empty dict will block all
this is mandatory

Reply:
Ack